omod/src/main/java/org/openmrs/module/aijar/web/controller/CheckPublicHolidayController.java

```diff
@@ -0,0 +1,41 @@
+package org.openmrs.module.aijar.web.controller;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.aijar.api.AijarService;
+import org.openmrs.module.aijar.metadata.core.PublicHolidays;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/ugandaemr/checkPublicHoliday")
```

Comment:
@reagan-meant instead of ugandaemr use the constant created UGANDAEMR_MODULE_ID

Reply:
Done